ui_framework/doc_site/src/views/form/color_picker.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import { KuiColorPicker } from '../../../../components/index';
+
+export class ColorPicker extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      color: null
+    };
+  }
+
+  handleChange = (value) => {
+    this.setState({ color: value });
+  };
+
+  render() {
+    return <KuiColorPicker onChange={ this.handleChange } color={ this.state.color }/>;
+  }
+}
```

Comment:
If we make the ColorPicker component live in its own directory, then can we also surface the ColorPicker example as its own page instead of part of the Form page?